python/requirements/data_processing/requirements.txt

```diff
@@ -8,7 +8,8 @@ crc32c==2.3
 flask_cors
 s3fs==2023.1.0
 modin==0.12.1; python_version < '3.8'
-modin==0.18.1; python_version >= '3.8'
+# Todo: Upgrade and fix pandas errors
+modin==0.12.1; python_version >= '3.8'
```

Comment:
Why do we differentiate between Python versions here if both groups pin at the same version?